src/main/scala/discord/model/DispatchEvent.scala

```diff
@@ -118,22 +173,22 @@ object DispatchEvent {
       data.as[Ready]
     case "RESUMED" =>
       Resumed.asRight
+    // TODO: We lose context here. CHANNEL_CREATE, CHANNEL_UPDATE, and CHANNEL_DELETE provide a
+    // TODO: Channel object but we don't know if it was created or updated or deleted anymore
     case n @ "CHANNEL_CREATE" =>
-      ImplementMe(n)
+      data.as[Channel]
     case n @ "CHANNEL_UPDATE" =>
-      ImplementMe(n)
+      data.as[Channel]
     case n @ "CHANNEL_DELETE" =>
-      ImplementMe(n)
+      data.as[Channel]
     case n @ "CHANNEL_PINS_UPDATE" =>
-      ImplementMe(n)
+      data.as[ChannelPinsUpdate]
```

Comment:
Do we still need `n`?

Reply:
Oh ya, good catch. I will get rid of those everywhere they aren't needed 👍